SQL_files/baseball_batting_average.sql

```diff
@@ -0,0 +1,52 @@
+HOW ERRORS;
+SHOW WARNINGS;
+COMMIT;
+
+/* 
+use baseball schema 
+Note: BA = avg(hit/atBat)
+*/
+USE baseball;
+
+/*
+Table: historical table (BA overall) 
+Note: COUNT(batter) = number of times batter played and number of games (batter shows up once per game)
+*/
+DROP TABLE IF EXISTS BATTERS_HISTORICAL;
+CREATE TABLE BATTERS_HISTORICAL AS
+(SELECT batter AS Batter, COUNT(batter) AS Batters_Count, SUM(Hit) AS Hit_Sum, SUM(atBat) AS atBat_Sum, SUM(Hit)/NULLIF(SUM(atBat), 0) AS Batting_AVG
```

Comment:
Is it necessary to COUNTY(batter) if we are also grouping by batter in the next line?

Reply:
No I just did that for myself.

Reply:
It's always a good idea to have a COUNT in there as it can help you debug issues.